examples/pydantic-ai/agent.py

```
@@ -0,0 +1,81 @@
+            "You are a helpful weather assistant. "
+            "Use the get_weather tool to fetch weather data for locations. "
+            "Always return responses in the required structured format."
+        ),
+    )
+
+    agent.tool(get_weather)
+    return agent
+
+
+async def run_weather_agent(
+    query: str, model: str = "openai:gpt-4.1-mini"
```

Ruff (0.11.9)

Trailing comma missing

Add trailing comma

(COM812)